sdk/rust/oak/src/lib.rs

```diff
@@ -19,6 +19,7 @@ use log::{debug, error};
 use protobuf::ProtobufEnum;
 
 // Re-export ABI constants that are also visible as part of the SDK API.
+pub use oak_abi::wasm;
```

Comment:
I'm not entirely convinced we should be re-exporting stuff, but let's discuss that separately later on.